test/stdlib/DispatchDeprecationWatchOS.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %swift -typecheck -target i386-apple-watchos2.0 -verify -sdk %sdk %s
-// REQUIRES: OS=watchos
+// REQUIRES: CPU=x86_64, OS=watchos
```

Comment:
```
18:00:42 UNSUPPORTED: Swift(watchsimulator-i386) :: stdlib/DispatchDeprecationWatchOS.swift (431 of 9456)
```

Looks like it's no good. It might just be that you need to use i386 instead of x86_64, but please test locally before merging!